bms/bms_app/static/bms_app/js/project_list.js

```diff
@@ -57,3 +57,10 @@ $(document).ready(function() {
     });
 });
 
+$(document).ready(function() {
```

Comment:
no need to use "+$(document).ready(function() {" each and every time if already used, you can put event in the existing one